src/main/java/org/broadinstitute/consent/http/db/MatchDAO.java

```diff
@@ -12,53 +12,54 @@
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 import org.jdbi.v3.sqlobject.transaction.Transactional;
 
-import java.util.Collection;
 import java.util.Date;
 import java.util.List;
 
 @RegisterRowMapper(MatchMapper.class)
 public interface MatchDAO extends Transactional<MatchDAO> {
 
-    @SqlQuery("select * from match_entity where consent = :consentId ")
-    List<Match> findMatchByConsentId(@Bind("consentId") String consentId);
+    @SqlQuery("SELECT * FROM match_entity WHERE consent = :consentId ")
+    List<Match> findMatchesByConsentId(@Bind("consentId") String consentId);
 
-    @SqlQuery("select * from match_entity where purpose = :purposeId ")
-    List<Match>  findMatchByPurposeId(@Bind("purposeId") String purposeId);
+    @SqlQuery("SELECT * FROM match_entity WHERE purpose = :purposeId ")
+    List<Match> findMatchesByPurposeId(@Bind("purposeId") String purposeId);
 
-    @SqlQuery("select * from match_entity where purpose = :purposeId and consent = :consentId ")
-    Match  findMatchByPurposeIdAndConsent(@Bind("purposeId") String purposeId, @Bind("consentId") String consentId);
+    @SqlQuery("SELECT * FROM match_entity WHERE purpose = :purposeId AND consent = :consentId ")
+    Match findMatchByPurposeIdAndConsentId(@Bind("purposeId") String purposeId, @Bind("consentId") String consentId);
 
-    @SqlQuery("select * from match_entity where matchId = :id ")
+    @SqlQuery("SELECT * FROM match_entity WHERE matchid = :id ")
     Match  findMatchById(@Bind("id") Integer id);
 
-    @SqlQuery("select * from match_entity where purpose  IN (<purposeId>)")
-    List<Match>  findMatchesPurposeId(@BindList("purposeId") List<String> purposeId);
+    @SqlQuery("SELECT * FROM match_entity WHERE purpose IN (<purposeId>)")
+    List<Match> findMatchesForPurposeIds(@BindList("purposeId") List<String> purposeId);
 
-    @SqlUpdate("insert into match_entity " +
-            "(consent, purpose, matchEntity, failed, date) values " +
-            "(:consentId, :purposeId, :match, :failed, :createDate)")
+    @SqlUpdate(
+            " INSERT INTO match_entity " +
+            " (consent, purpose, matchentity, failed, createdate) VALUES " +
+            " (:consentId, :purposeId, :match, :failed, :createDate)")
```

Comment:
This was broken because we do not have a `date` field, we have a `createdate` field.